### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by krukelena**
add HW2 task1,2,3

### CODE REVIEW

File: HomeWork_2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            Console.Write("Введите второе число: ");
+            decimal operand2 = decimal.Parse(Console.ReadLine());
+
+            Console.Write("Введите знак арифметической операции число: ");
+            string sign = Console.ReadLine();
+
+            switch (sign)
+            {
+                case "+":
+                    {
```
Comment (VladimirVolkovVD) [inline HomeWork_2/Program.cs]: Don't need figure brackets

File: HomeWork_2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        //Задайте переменным некоторые произвольные значения.
+        //Предложите пользователю ввести знак арифметической операции.
+        //Примите значение, введенное пользователем, и поместите его в строковую переменную sign.
+        //Для организации выбора алгоритма вычислительного процесса, используйте переключатель switch.
+        //Выведите на экран результат выполнения арифметической операции.
+        //**В случае использования операции деления, организуйте проверку попытки деления на ноль.
+        //И если таковая имеется, то отмените выполнение арифметической операции и уведомите об ошибке пользователя.
+        static void Task1()
+        {
+            Console.Write("Введите первое число: ");
```
Comment (VladimirVolkovVD) [inline HomeWork_2/Program.cs:29]: Russian text

File: HomeWork_2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                Console.WriteLine($"Число от 50 до 100");
+            }
+            else
+            {
+                Console.WriteLine($"Не соответсвует числовым промежуткам");
+             }
+
+
+
+
```
Comment (VladimirVolkovVD) [inline HomeWork_2/Program.cs]: extra lines

**Review by VladimirVolkovVD — COMMENTED** [review]
cosmetic changes need

File: HomeWork_2/Program.cs (hunk 2 above)
Comment (krukelena) [reply HomeWork_2/Program.cs:29]: Пока мне удобно использовать русский текст.

File: HomeWork_2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+            }
+            else if ((number >= 50) && (number <= 100))
+            {
+                Console.WriteLine($"Число от 50 до 100");
+            }
+            else
+            {
+                Console.WriteLine($"Не соответсвует числовым промежуткам");
+            }
+        }
```
Comment (VladimirVolkovVD) [inline HomeWork_2/Program.cs:94]: Add empty line after method

File: HomeWork_2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                        Console.WriteLine("Делить на 0 нельзя");
+                    else
+                        Console.WriteLine(operand1 / operand2);
+                    break;
+                default:
+                    Console.WriteLine("Неизвестное математическое действие");
+                    break;
+
+            }
+        }
```
Comment (VladimirVolkovVD) [inline HomeWork_2/Program.cs:61]: Add empty line after method

**Review by VladimirVolkovVD — APPROVED** [review]
Add empty line after method and can merge

**Pull request merged**